### DISCUSSION TITLE
cannot import name '_get_height_ratios'

### DISCUSSION THREAD

**Discussion by BenedettaFrancesconi** [pastas/pastas#440] [Q&A] [answered]
Hello everyone,

I am trying to install metran, for which pastas is one of the packages require. Every time that I try to import metran, I keep getting the error 'cannot import name '_get_height_ratios' from pastas.plot' and in fact, checking the plot.py code I can't find any function called _get_height_ratios . Has anyone already had this problem and has been able to solve it? Thank you!

**Comment by martinvonk** [comment]
Hi @BenedettaFrancesconi,

This is a known issue that has been resolved by [https://github.com/pastas/metran/pull/7](https://github.com/pastas/metran/pull/7), but is not "live" yet. From Pastas version 0.19.0 onwards, `_get_heights_ratios` was moved to `pastas.modelplots`. So to resolve the issue quickly, you could downgrade Pastas to a version below 0.19.0. However, a better and also quick solution would be to run `metran` from the [dev branch](https://github.com/pastas/metran/tree/dev). Cloning the branch and using `pip install -e .` should do the trick, or you could use `pip install git+https://github.com/pastas/metran.git@dev#egg=metran`. 

Additionally, we could ask @dbrakenhoff to do a Christmas release of Metran :)

Hope this helps!

**Comment by dbrakenhoff** [comment]
Christmas release sounds good!

**Comment by martinvonk** [comment, marked as answer]
Hi @BenedettaFrancesconi,

There is now a new version of Metran ([0.2.0](https://github.com/pastas/metran/releases/tag/v0.2.0)) available that has this issue resolved. 
You can install it using `pip install metran --upgrade`
